test/integration/options.spec.js

```diff
@@ -280,4 +282,110 @@ describe('options', function () {
       });
     });
   });
+
+  describe('--no-exit', function () {
```

Comment:
I'm not sure we need this, as it seems to pass with or without the changes in `runnable.js`.  Either way I'm trying to combine it with what we have for `--exit` tests, as I wrote for v4.